packages/ra-core/src/form/Form.tsx

```diff
@@ -50,7 +50,10 @@ export const Form = (props: FormProps) => {
 
     return (
         <OptionalRecordContextProvider value={record}>
-            <LabelPrefixContextProvider prefix={`resources.${resource}.fields`}>
+            <LabelPrefixContextProvider
+                prefix={`resources.${resource}.fields`}
+                concatenate={false}
+            >
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:** ‏We probably need to do the same thing in `FilterForm.tsx`